tests/ops/qubit/test_sparse.py

```diff
@@ -120,7 +120,7 @@ def circuit(param):
             match="SparseHamiltonian observable must be"
             " used with the parameter-shift differentiation method",
         ):
-            qml.grad(circuit, argnum=0)([0.5])
+            qml.grad(circuit, argnum=0)(0.5)
```

Comment:
How is it related? I see that it is batching where we do not need it but it should run and work

Reply:
was fixed by a later change.